src/stories/containers/RecognizedDelegates/KeyStats/KeyDescriptionCards/VisualizationCard/VisualizationCard.tsx

```diff
@@ -0,0 +1,92 @@
+import styled from '@emotion/styled';
+import React from 'react';
+import KeyStatsCard from '../../../GenericDelegateCard';
+import LegendItem from './LegendItem';
+import { RelativeDelegateBar } from './RelativeDelegateBar';
+
+interface Props {
+  percent: number;
+  totalDai: number;
+}
+
+const VisualizationCard: React.FC<Props> = ({ percent }) => (
+  <ExtendedKeyStatsCard>
+    <Legend>
+      <LegendItem color="#ECF1F3" description="Other Expenses" />
+      <LegendItem color="#447AFB" description="Recognized Delegates (21)" />
```

Comment:
The `21` looks like a props for me, the amount of existing delegates

Reply:
you are right, this is the number of recognized delegates